.circleci/real_config.yml

```diff
@@ -13,15 +13,12 @@ orbs:
 executors:
   python-38:
     docker:
-      - image: python:3.8-slim-bookworm
```

Comment:
these didnt have git installed, so they couldn't run the script

Reply:
Good catch!